src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -328,6 +343,20 @@ export class DictionaryModal extends React.Component {
                       value="OpenMRSDictionary"
                     />
                   </FormGroup>
+                  {!isEditingDictionary &&
+                  <FormGroup>
+                    {'Start by copying another dictionary'}
+                    <Select
+                      id="copy_dictionary"
+                      closeMenuOnSelect={false}
+                      isLoading={this.props.isLoadingDictionaries}
+                      options={[...this.props.userDictionaries, ...this.props.dictionaries]}
+                      onChange={val => this.handleCopyDictionary(val)}
+                      onInputChange={(val) => { this.searchInputValues(val); }}
```

Comment:
Yes, it is. The function runs whenever there is an onchange event. If I remove the braces, it just returns the functions causing the application to break because the component I used does some inner text processing which will cause the application to break.

Reply:
I have made the update